README.md

```diff
@@ -31,4 +34,7 @@ Access and use of the dataset are subject to the following [Terms & Conditions](
 |hm3d-example-glb.tar**|example|glb|[hm3d-example-glb.tar](example/hm3d-example-glb.tar)|186M|
 |hm3d-example-habitat.tar**|example|habitat|[hm3d-example-habitat.tar](example/hm3d-example-habitat.tar)|155M|
 |hm3d-example-obj+mtl.tar.gz|example|obj+mtl|[hm3d-example-obj+mtl.tar.gz](example/hm3d-example-obj+mtl.tar.gz)|179M|
+|🆕 hm3d-train-semantic-annots-v0.1.tar.gz|train|habitat|https://api.matterport.com/resources/habitat/hm3d-train-semantic-annots-v0.1.tar.gz|1.9G|
+|🆕 hm3d-val-semantic-annots-v0.1.tar.gz|val|habitat|https://api.matterport.com/resources/habitat/hm3d-val-semantic-annots-v0.1.tar.gz|433M|
```

Comment:
Does -test- have the same filename format? If so, someone can just change the filename and download test? 

Have we confirmed that Matterport mark the test files not downloadable?

Reply:
I've requested Matterport folks to set permissions accordingly. From what @erikwijmans mentioned, it looks like all these files are not readable by default.